src/resolveGitDir.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
-const findParentDir = require('find-parent-dir')
+const execa = require('execa')
 
-module.exports = function resolveGitDir() {
-  return findParentDir.sync(process.cwd(), '.git')
+module.exports = async function resolveGitDir() {
+  return (await execa('git', ['rev-parse', '--show-toplevel'])).stdout
```

Comment:
Do you think we could instead use the git command runner I wrote. It will simplify it a bit. 

https://github.com/okonet/lint-staged/blob/315890a31162c70398b7f467bc5d2c64ed391b3c/src/gitWorkflow.js#L18

Also this function also uses the old bit you refactoring. And the error handling is missing now from your implementation.

Reply:
Thanks for the comment. I extracted the `execGit` function into its own file, and used it here.

Reply:
> I like the proposed change but please check my comments. Let’s update Windows tests.

I was able to prevent changes to the windows git path by wrapping it in `path.normalize` here:

```suggestion
  const gitDir = await execGit(['rev-parse', '--show-toplevel'], options)
  return path.normalize(gitDir)
```